src/translator/quality_estimator_helper.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <memory>
+
+#include "definitions.h"
+#include "iquality_estimator.h"
+
+namespace marian::bergamot {
+
+/// The createQualityEstimator method create a quality estimator
+/// By default, if the qualityFileMemory is empty it will use
+/// the unsupervised learning approach (UnsupervisedQE).
+std::shared_ptr<IQualityEstimator> createQualityEstimator(const AlignedMemory& qualityFileMemory);
```

Comment:
We unified all the classes in the `quality_estimator.<cpp|h>` . 

> so cut unit tests that don't use this path - the cmd supplied tests through BRT is necessary and sufficient).

We decided to keep the unit test because we would like to assure that the functions were working as expected (i.e. inputs and outputs were the same as the python code ). Indeed BRT runs this part of the code, but it assumes that all the code "pieces" are working properly, which it's a different type of test since BRT validates the e2e flow. WDYT? 

In any case here is the commit with the major changes. 2403353